File: Python/Course.py
# deklarasi kelas course
class Course:

    # atribut private milik kelas course
    __nama_matkul = ""

    # constructor
    def __init__(self, nama_matkul=""):
        self.__nama_matkul = nama_matkul

    # setter nama matkul
    def setNamaMatkul(self, nama_matkul):
        self.__nama_matkul = nama_matkul

    # getter nama matkul
    def getNamaMatkul(self):
        return self.__nama_matkul

File: CPP/Course.cpp
#include "header.hh"

/* Definisi Method Kelas Course */

// constructor kosong
Course::Course()
{
}

// constructor dengan parameter
Course::Course(string nama_matkul)
{
    this->nama_matkul = nama_matkul;
}

// Setter dan Getter atribut kelas Course //

void Course::setNamaMatkul(string nama_matkul)
{
    this->nama_matkul = nama_matkul;
}

string Course::getNamaMatkul()
{
    return this->nama_matkul;
}

// destructor
Course::~Course()
{
}